modules/mobile/app/models/mobile/v0/adapters/decision_letters.rb

```diff
@@ -4,9 +4,14 @@ module Mobile
   module V0
     module Adapters
       class DecisionLetters
+        # rubocop:disable Metrics/MethodLength
```

Comment:
Rather than make a rubocop exception, I'd suggest we just move the `Mobile::V0::DecisionLetter.new` line into a separate method. It's the reason this method is getting too long.